### PR TITLE
BOM-1022

### PR DESCRIPTION
**Pull request by awais786**
Adding tox for django20,21,22.
https://openedx.atlassian.net/browse/BOM-1022

**Description:** Describe in a couple of sentence what this PR adds

**JIRA:** Link to JIRA ticket

**Dependencies:** dependencies on other outstanding PRs, issues, etc. 

**Installation instructions:** List any non-trivial installation 
instructions.

**Testing instructions:** List testing instructions

**Reviewers:**
- [ ] tag reviewer 
- [ ] tag reviewer 

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Version bumped
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPi after tag-triggered build is 
      finished.
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

File: requirements/base.in [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Core requirements for using this application
 -c constraints.txt
 
-Django>=1.8,<2.0          # Web application framework
+Django>=1.8          # Web application framework
```
Comment (Ayub-Khan) [inline requirements/base.in]: 1.11

File: setup.py [hunk 2]
```diff
@@ -100,5 +100,6 @@ def load_requirements(*requirements_paths):
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
```
Comment (Ayub-Khan) [inline setup.py]: I think we need Django and DRF support classifiers.

Comment (awais786) [reply setup.py]: I have aded django and i think its enough

File: requirements/doc.txt [hunk 3]
```diff
@@ -4,11 +4,9 @@
 #
 #    make upgrade
 #
-alabaster==0.7.12         # via sphinx
```
[the remark this answers is not in the document]
Comment (awais786) [reply requirements/doc.txt]: @jmbowman it worked.

File: .travis.yml [hunk 4]
```diff
@@ -1,22 +1,59 @@
 language: python
 python:
 - 2.7
+- 3.5
 - 3.6
```
Comment (jmbowman) [inline .travis.yml]: Maybe skip the 3.6 tests for now?  I really don't want to use 38 Travis workers for each PR on this repo, 22 is slightly more reasonable.

**Pull request merged**